bin/setup_psqlgraph.py

```diff
@@ -11,7 +11,7 @@ def try_drop_test_data(user, database, root_user="postgres", host=""):
     print("Dropping old test data")
 
     engine = create_engine(
-        "postgres://{user}@{host}/postgres".format(user=root_user, host=host)
+        "postgresql://{user}@{host}/postgres".format(user=root_user, host=host)
```

Comment:
does it make any difference? just curious 😄

Reply:
Either DSN should work. I could've sworn I saw an error from one of our packages while working on this PR, but now I don't see it. I reverted my change here.